config/main.py

```diff
@@ -7341,6 +7343,9 @@ def add_subinterface(ctx, subinterface_name, vid):
 
         if interface_alias is None:
             ctx.fail("{} invalid subinterface".format(interface_alias))
+        
+        if len(subinterface_name) >= IFNAMSIZ:
```

Comment:
@arfeigin is this a typo? What should be `>=` or `>`?

Reply:
>=
As it should not exceed this length including terminating zero.